src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
> I hate this everyone-should-follow-the-same-style mentality

I'm saying this project needs to be consistent with other projects in the php league.